packages/web/src/pages/track-page/components/desktop/TrackPage.tsx

```diff
@@ -150,13 +154,17 @@ const TrackPage = ({
   }
 
   const defaults = getTrackDefaults(heroTrack)
+  const trackPositionInfo = useSelector((state: CommonState) =>
+    getTrackPosition(state, { trackId: heroTrack?.track_id })
+  )
 
   const renderGiantTrackTile = () => (
     <GiantTrackTile
       loading={loading}
       playing={heroPlaying}
       trackTitle={defaults.title}
       trackId={defaults.trackId}
+      playbackPositionInfo={trackPositionInfo}
```

Comment:
just to close the loop on this one, ideally we dont have to continue to add to a components props as new features get added, ya know?